src/Shared/ResourceDetectorUtils.cs

```diff
@@ -12,7 +12,7 @@
 #endif
 using System.Threading.Tasks;
 
-namespace OpenTelemetry.Resources.AWS;
```

Comment:
In `ResourceDetectorUtils.SendOutRequest` one-time used `HttpClient` can be created and not disposed.
It would be great to fix it in future to something like:
```
using var httpClient = handler == null ? new HttpClient() : new HttpClient(handler, false);
```

Reply:
Yes, I've already planned to rework `ResourceDetectorUtils` in a future PR as agreed above (in particular to avoid sync-over-async).